.claude/scripts/mcp_common.sh

```
@@ -0,0 +1,2130 @@
+
+                local dep_exit_code=0
+                if [ -f "${REPO_ROOT}/react-mcp/package-lock.json" ]; then
+                    capture_command_output dep_output dep_exit_code npm --prefix "${REPO_ROOT}/react-mcp" ci
+                else
+                    echo -e "${YELLOW}  ⚠️ No package-lock.json found, using npm install instead${NC}"
+                    capture_command_output dep_output dep_exit_code npm --prefix "${REPO_ROOT}/react-mcp" install
+                fi
+
+                if [ $dep_exit_code -ne 0 ]; then
+                    echo -e "${RED}  ❌ Failed to install react-mcp dependencies${NC}"
+                    log_error_details "npm dependency installation (react-mcp)" "react-mcp" "$dep_output"
```

Shellcheck (0.11.0)

[warning] dep_output is referenced but not assigned.

(SC2154)